cairo/ethereum/cancun/vm/precompiled_contracts/identity.cairo

```diff
@@ -0,0 +1,34 @@
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin, KeccakBuiltin
+from starkware.cairo.common.math_cmp import is_le_felt
+from ethereum.cancun.vm import Evm, EvmImpl
+from ethereum.exceptions import EthereumException
+from ethereum.cancun.vm.exceptions import OutOfGasError
+from ethereum.utils.numeric import ceil32
+from ethereum.cancun.vm.gas import GasConstants, charge_gas
+from ethereum_types.numeric import Uint
+
+// @notice Writes the message data to the output
+func identity{range_check_ptr, bitwise_ptr: BitwiseBuiltin*, keccak_ptr: KeccakBuiltin*, evm: Evm}(
+    ) -> EthereumException* {
+    let data = evm.value.message.value.data;
+
+    // Gas
+    let ceiled_words = ceil32(Uint(data.value.len));
+    let word_count = ceiled_words.value / 32;  // simple div as we're dividing a number divisible by 32
```

Comment:
very good point.
Some part of the codebase are not taking advantage of this and use divmod 
Followup issue https://github.com/kkrt-labs/keth/issues/613